lib/prompts/createModulePrompt.ts

```diff
@@ -30,10 +30,18 @@ const CONTENT_TYPES_PROMPT = {
   message: i18n(`${i18nKey}.selectContentType`),
   default: ['PAGE'],
```

Comment:
Hi @kotto5 - one quick note that we should update the default here to point to the `ANY` option instead to support this change

Reply:
Thanks for review @TanyaScales 

I fixed it based on your point, and I'm glad to see that the UI thing is not a problem either.
Hope you will check the fixes.

Reply:
@TanyaScales 
How do I ask other reviewers for a review after I have been approved by you?